style.css

```diff
@@ -0,0 +1,733 @@
+/* stylelint-disable no-descending-specificity */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #f7f7f9;
+}
+
+header {
+  font-family: 'Poppins', sans-serif;
+  display: flex;
+  justify-content: space-between;
+  padding: 1em 2em;
+  float: left;
+  position: fixed;
+  top: 10px;
+  width: 100%;
+  z-index: 100;
+}
+
+header .logo {
+  display: inline-block;
+  font-family: 'Poppins', sans-serif;
+  font-size: 22px;
+  font-weight: 700;
+  color: #6070ff;
+  text-shadow: 1px 1px 0.6px #091e42;
+  line-height: 20px;
+  text-align: center;
+  cursor: pointer;
+}
+
+header .menuicon {
+  cursor: pointer;
+}
+
+header .menuicon span {
+  display: block;
+  width: 25px;
+  height: 4px;
+  background: #6070ff;
+  margin-bottom: 5px;
+}
+
+header .navbar {
+  justify-content: space-evenly;
+  align-items: center;
+  display: none;
+}
+
+header .navbar a {
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  color: #344563;
+  cursor: pointer;
+  text-decoration: none;
+  margin: auto 1em;
+}
+
+.navbar a:hover {
+  color: #6070ff;
+  text-shadow: 1px 1px 0.6px #091e42;
+  transition: all 0.5s;
+}
+
+/* mobile view nav icon click  */
+.navheader {
+  height: 100vh;
+  background-color: rgba(96, 112, 255, 0.9);
+  top: 0;
+  flex-wrap: wrap;
+}
+
+.navheader .logo {
+  visibility: hidden;
+}
+
+.navheader .menuicon {
+  padding-top: 0.5em;
+  height: 30px;
+}
+
+.navheader .menuicon span {
+  display: block;
+  width: 25px;
+  height: 4px;
+  background: #fff;
+  margin-bottom: 5px;
+  transition: transform 0.5s;
+}
+
+/* making cross icon  */
+.navheader .menuicon span:nth-child(1) {
+  transform: rotate(45deg) translateX(50%);
+}
+
+.navheader .menuicon span:nth-child(2) {
+  visibility: hidden;
+}
+
+.navheader .menuicon span:nth-child(3) {
+  transform: rotate(-45deg) translateX(50%);
+}
+
+/* end of cross icon  */
+
+.navheader .navbar {
+  display: flex;
+  width: 100%;
+  margin-top: -100px;
+  text-align: center;
+}
+
+.navheader .navbar a {
+  display: block;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 1em;
+  color: #fff;
+}
+
+.navheader .navbar a:hover {
+  color: black;
+}
+
+/* End of header section  */
+
+/* Start of  home page section */
+
+.homepage,
```

Comment:
Kindly consider adding a background image to 'homepage'  to make it look like the provided Figma design 😁. Since we are practicing our skills for transforming design into code, following the design as much as possible in this project would be important 💯.

Left: Figma Design;   Right: Your Design 

![Screenshot 2023-11-25 120034](https://github.com/HsuMyatPhoo/PortfolioFinishMobileVersion/assets/102011195/c7da8cab-d823-45e7-a4f4-61d551b46bb3)